server/services/config/crm5DisplayConfig.json

```diff
@@ -278,106 +278,106 @@
       ]
     },
     {
-      "sectionId": "advice-and-assistance",
-      "title": "Advice and Assistance",
+      "sectionId": "proceedings",
```

Comment:
_"Moved Proceedings, Statement of Case into the correct order in display config file"_